### DISCUSSION TITLE
v0.18.6

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1365] [Releases]
## What's New
* Add upload and download latency information by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1353
* Update RecentDownloadLatencyChartWidget.php by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1356
* [Feature] Added ability to truncate the results table by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1362

## What's Changed
* Bump vite from 5.1.6 to 5.1.7 by @dependabot in https://github.com/alexjustesen/speedtest-tracker/pull/1348
* Feature/result hascolor by @wdog in https://github.com/alexjustesen/speedtest-tracker/pull/1347
* Laravel 10.48.5 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1361
* [Chore] Badges labels and color updates by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1363
* Release v0.18.6 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1364

## New Contributors
* @svenvg93 made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/1353

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.18.5...v0.18.6

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.18.6'>v0.18.6</a>.</em>

**Comment by sobeservices** [comment]
Does this auto update? I am still stuck on 18.5

**Reply by alexjustesen** [reply]
The container doesn't update itself.

**Reply by sobeservices** [reply]
How would you update it then? I kind of been using Portainer as well.

**Reply by alexjustesen** [reply]
🤷‍♂️ portainer would of. 

The app doesn't update itself it would be whatever you use to manage your containers.

**Reply by seh2000** [reply]
Edit your docker-compose.yml file and then do 

docker-compose up -d or docker-compose up -d && docker-compose logs -f

Check `https://docs.speedtest-tracker.dev` for more...

**Reply by sobeservices** [reply]
Could you show me an example of a proper docker-compose file? I'm sorry, but I'm very new and learning quickly.

**Reply by seh2000** [reply]
In the documents you will find (linked above) 
` https://docs.speedtest-tracker.dev/getting-started/installation/installation `

**Reply by sobeservices** [reply]
I see that but was hoping someone would post a copy of theirs so I can copy from. I really don't see an example in the guids on what to use or how to format it correctly unless I am completely missing something. Sorry 

Is this what the file should look like?

docker run -d --name speedtest-tracker --restart unless-stopped \
    -p 8080:80 \
    -p 8443:443 \
    -e PUID=1000 \
    -e PGID=1000 \
    -e DB_CONNECTION=sqlite \
    -v /path/to/data:/config \
    -v /path/to-custom-ssl-keys:/config/keys \
    lscr.io/linuxserver/speedtest-tracker:latest

Confused a bit

**Reply by seh2000** [reply]
The linked document shows in clear details how to do the install.
Which one to use depends on the DB to use and if you use Docker or Docker Compose.

![image](https://github.com/alexjustesen/speedtest-tracker/assets/74658646/f1c026cd-95a1-459d-9b30-ab27b99c5b36)

File you posted is for Docker, I use Docker Compose as recommended in the document.

Individual changes are related to the individual instance one uses e.g. username/passwords.

There are as well two videos that Alex have included in the documentation, check first page "Introduction".

The below is a copy from the document for MariaDB/MySQL, for me it worked as it is. 

version: '3.4'
services:
    speedtest-tracker:
        container_name: speedtest-tracker
        ports:
            - 8080:80
            - 8443:443
        environment:
            - PUID=1000
            - PGID=1000
            - DB_CONNECTION=mysql
            - DB_HOST=db
            - DB_PORT=3306
            - DB_DATABASE=speedtest_tracker
            - DB_USERNAME=speedy
            - DB_PASSWORD=password
        volumes:
            - /path/to/data:/config
            - /path/to-custom-ssl-keys:/config/keys
        image: lscr.io/linuxserver/speedtest-tracker:latest
        restart: unless-stopped
        depends_on:
            - db
    db:
        image: mariadb:10
        restart: always
        environment:
            - MARIADB_DATABASE=speedtest_tracker
            - MARIADB_USER=speedy
            - MARIADB_PASSWORD=password
            - MARIADB_RANDOM_ROOT_PASSWORD=true
        volumes:
            - speedtest-db:/var/lib/mysql
volumes:
  speedtest-db:
